pkg/pgmodel/migrations/generate.go

```diff
@@ -9,11 +9,12 @@
 package main
 
 import (
-	"github.com/shurcooL/vfsgen"
 	"log"
 	"net/http"
 	"os"
 	"time"
+
+	"github.com/shurcooL/vfsgen"
```

Comment:
```suggestion
```

Reply:
Why remove this? Its used to generate the virtual filesystem for migration files.